### PR TITLE
You had a HEAD tag showing up at the top of the file. 

### PR DESCRIPTION
**Pull request by naluconcepcion**
Not sure if this was intentional, but I commented the lines out for now.

### CODE REVIEW

**Comment by ruddfawcett** [comment]
Definitely not intentional — could you remove them in a new commit (sorry not at a computer right now)?

File: essays.html [hunk 1]
```diff
@@ -3,11 +3,11 @@
 <head>
     <meta charset="utf-8" >
     <title>CSC630: The Open Source Movement</title>
-<<<<<<< HEAD
+<!<<<<<< HEAD >
     <link rel="stylesheet" type="text/css" href="./assets/styles/main.css" />
-=======
+<!=======>
     <link rel="stylesheet" type="text/css" href='./assets/styles/main.css' />
```
Comment (milesmcc) [inline essays.html:9]: This code seems identical to line 7—perhaps this should be removed as well?

**Review by milesmcc — CHANGES_REQUESTED** [review]
This is definitely a residual merge conflict. Perhaps line 9 should also be removed so that there are not duplicate imports?

**Comment by nzufelt** [comment]
I think this is resolved by #84.  Closing for now, reopen if needed.

**Pull request closed without merging**